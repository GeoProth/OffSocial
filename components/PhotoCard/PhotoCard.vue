<template>
  <div class="photo-card">
    <div class="img" :style="{ backgroundImage: `url(${url})` }" />
    <DownloadButton :url="url" />
  </div>
</template>
<script>
export default {
  name: "PhotoCard",
  props: {
    url: String
  }
};
</script>
<style lang="stylus" scoped>
.photo-card {
  height: fit-content;
  width: 90%;
  border-radius: 4px;
  background-color: ui_gray;
  box-shadow: -2px 2px 12px -1px;
  margin-top: 2rem;
  margin-bottom: 2rem;
  margin-left: auto;
  margin-right: auto;
  &:hover {
    box-shadow: -5px 5px 12px -1px;
    transform: scale(1.025, 1.025);
  }
  .img {
    height: 20rem;
    background-position: top;
    border-radius: 4px;
  }

}
</style>
