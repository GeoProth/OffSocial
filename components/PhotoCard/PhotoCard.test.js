import PhotoCard from "./PhotoCard";
describe("PhotoCard", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(PhotoCard);
  });
  test("Renders", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
