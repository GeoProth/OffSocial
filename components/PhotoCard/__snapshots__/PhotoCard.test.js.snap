// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoCard Renders 1`] = `
<div class="photo-card">
  <div class="img" style="background-image: url(undefined);"></div>
  <downloadbutton-stub text="Download"></downloadbutton-stub>
</div>
`;
